schemas/2019-08-01-preview/Microsoft.OperationalInsights.json

```diff
@@ -46,21 +56,21 @@
               "$ref": "#/definitions/Identity"
             },
             {
-              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
             }
           ],
-          "description": "The identity of the resource."
+          "description": "Identity for the resource."
         },
         "properties": {
           "oneOf": [
             {
               "$ref": "#/definitions/ClusterProperties"
             },
             {
-              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
             }
           ],
-          "description": "Log Analytics cluster properties."
```

Comment:
Added 'Log Analytics' to make it clearer. Is there any way to add it back?

Reply:
It would require a change to the [swagger spec](https://github.com/Azure/azure-rest-api-specs/blob/454b15b1a325e019606c16d96350440f7abd6d0c/specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2019-08-01-preview/Clusters.json#L378) for this resource. Note that to get this intellisense suggestion in the first place, the customer would have to create the following minimum, so the RP namespace and resource type are already implicit:
```
{
    "type": "Microsoft.OperationalInsights/clusters",
    "apiVersion": "2019-08-01-preview",
    "properties": <ctrl+space>
}
```

I'm also not clear why this would be "Log Analytics" related as this is the config for the OperationalInsights cluster resource. Are "Log Analytics" and "Operational Insights" analogous to the customer?

Reply:
Basically yes. OperationalInsights represents RP for Log Analytics workspaces and clusters.